driver_test.go

```
@@ -147,38 +147,58 @@ func runTests(t *testing.T, dsn string, tests ...func(dbt *DBTest)) {
+	}
+	defer db3.Close()
+
+	cleanupSql := "DROP TABLE IF EXISTS test"
+
 	for _, test := range tests {
 		test := test
 		t.Run("default", func(t *testing.T) {
 			dbt := &DBTest{t, db}
-			t.Cleanup(cleanup)
+			t.Cleanup(func() {
+				db.Exec(cleanupSql)
```

golangci-lint (1.62.2)

Error return value of `db.Exec` is not checked

(errcheck)